tools/postinstall

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -eu
+
+ROOT_DIR=$(git rev-parse --show-toplevel)
+
+pod_install() {
+    [[ "$OSTYPE" == "darwin"* ]] \
+        || return 0
+    hash pod 2>/dev/null \
+        || {
+            cat >&2 <<EOF
```

Comment:
formatting: when it's going to be a whole `{ ... }` sequence like this, use `if ... then ... fi` instead. In particular that keeps the body just one level nested.

So e.g.:
```shell
if ! hash pod 2>/dev/null; then
    cat # ...
    # ...
fi
```